samples/apply-scene-property-expressions/src/main/java/com/esri/arcgismaps/sample/applyscenepropertyexpressions/screens/ApplyScenePropertyExpressionsScreen.kt

```diff
@@ -0,0 +1,182 @@
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Settings
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.FloatingActionButton
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.ModalBottomSheet
+import androidx.compose.material3.OutlinedButton
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Slider
+import androidx.compose.material3.Text
+import androidx.compose.material3.rememberModalBottomSheetState
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.arcgismaps.toolkit.geoviewcompose.SceneView
+import com.esri.arcgismaps.sample.applyscenepropertyexpressions.components.ApplyScenePropertyExpressionsViewModel
+import com.esri.arcgismaps.sample.sampleslib.components.MessageDialog
+import com.esri.arcgismaps.sample.sampleslib.components.SamplePreviewSurface
+import com.esri.arcgismaps.sample.sampleslib.components.SampleTopAppBar
+
+/**
+ * Main screen layout for the sample app
+ */
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun ApplyScenePropertyExpressionsScreen(sampleName: String) {
+    val sceneViewModel: ApplyScenePropertyExpressionsViewModel = viewModel()
+    val sheetState = rememberModalBottomSheetState(skipPartiallyExpanded = true)
+    var isSheetOpen by remember { mutableStateOf(false) }
+
+    Scaffold(
+        topBar = { SampleTopAppBar(title = sampleName) },
+        content = { padding ->
+            Box(
+                modifier = Modifier
+                    .fillMaxSize()
+                    .padding(padding)
+            ) {
+                SceneView(
+                    modifier = Modifier.fillMaxSize(),
+                    arcGISScene = sceneViewModel.arcGISScene,
+                    graphicsOverlays = listOf(sceneViewModel.graphicsOverlay),
+                    sceneViewProxy = sceneViewModel.sceneViewProxy,
+                    onDown = {
+                        // Dismiss sheet when user interacts with the scene
+                        isSheetOpen = false
+                    }
+                )
```

Comment:
```suggestion
                    sceneViewProxy = sceneViewModel.sceneViewProxy
                )
```

Get this behaviour for free without it